RTCPeerConnection.js

```diff
@@ -206,6 +200,9 @@ export default class RTCPeerConnection extends EventTarget(PEER_CONNECTION_EVENT
             // general, the stress is on being faster to pass through the React
             // Native bridge which is a bottleneck that tends to be visible in
             // the UI when there is congestion involving UI-related passing.
+            if (Array.isArray(stats) && stats.length === 1 && typeof stats[0] === 'string') {
```

Comment:
why is that necessary ? is it possible to have something else than a JSON string currently ? if not then maybe we should check if it's a string and throw an error

Reply:
Good catch. I believe iOS returns (or used to) an array of a single element instead of the actual stuff. I plan to switch to the non-deprecated stats API and I'll take care of this then.

Reply:
so we want this as-is for now ?

Reply:
btw. there are some conflicts

Reply:
I'd say so, yeah. What conflicts?